eosio.system/abi/eosio.system.abi

```diff
@@ -240,7 +240,14 @@
          {"name":"max_inline_action_size",              "type":"uint32"},
          {"name":"max_inline_action_depth",             "type":"uint16"},
          {"name":"max_authority_depth",                 "type":"uint16"}
-
+      ]
+    },{
+      "name": "eosio_global_state2",
+      "fields": [
+         {"name":"new_ram_per_block",   "type":"uint16"},
+         {"name":"last_ram_increase",   "type":"block_timestamp"},
```

Comment:
timestamp field names could be more descriptive as well as the `state2` naming. Introduces a naming pattern that could be misappropriated later.